Info.lua

```diff
@@ -2,12 +2,13 @@ return {
     LrSdkVersion = 3.0,
     LrSdkMinimumVersion = 3.0, -- minimum SDK version required by this plug-in
     LrToolkitIdentifier = 'au.id.willthames.photodeck',
-    LrPluginName = 'PhotoDeck Publisher',
+    LrPluginName = 'PhotoDeck',
```

Comment:
Will this change cause problems for people using existing plugin? Bit reluctant to cause breaking changes for long term users.